packages/babel-preset-expo/src/web-preset.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright © 2024 650 Industries.
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * A fork of `@react-native/babel-preset` but with everything unrelated to web/ssr removed.
+ * https://github.com/facebook/react-native/blob/2af1da42ff517232f1309efed7565fe9ddbbac77/packages/react-native-babel-preset/src/configs/main.js#L1
+ */
+
+// use `this.foo = bar` instead of `this.defineProperty('foo', ...)`
+const loose = true;
+
+const defaultPlugins = [
+  [require('@babel/plugin-syntax-flow')],
+  [require('babel-plugin-transform-flow-enums')],
+  [require('@babel/plugin-proposal-class-properties'), { loose }],
+  [require('@babel/plugin-transform-private-methods'), { loose }],
+  [require('@babel/plugin-transform-private-property-in-object'), { loose }],
+  [require('@babel/plugin-syntax-dynamic-import')],
+  [require('@babel/plugin-syntax-export-default-from')],
+  [require('@babel/plugin-syntax-nullish-coalescing-operator')],
+  [require('@babel/plugin-syntax-optional-chaining')],
```

Comment:
Are these still needed? Given that this PR removes the transform for `??=`, I'd imagine that `?.` and `??` don't need the syntax plugins either. Or `@babel/plugin-syntax-logical-assignment-operators` is needed here. But ideally Babel would parse modern ratified JS without the plugins.

Reply:
These don't appear to be needed anymore.